Bug Report: "add-opens=java.base/java.io" is missing in Micro 6.2023.9/ FISH-7817

### Brief Summary

Your fix https://github.com/payara/Payara/pull/6371 does not work for Payara Micro.
You need to add it into MANIFEST.MF too.

This is what MANIFEST.MF in Micro 6.2023.9 looks like:

```
Manifest-Version: 1.0
Ant-Version: Apache Ant 1.10.12
Created-By: 11.0.14.1+1-LTS (Azul Systems, Inc.)
Bundle-SymbolicName: fish.payara.micro
Main-Class: fish.payara.micro.PayaraMicro
Start-Class: fish.payara.micro.impl.PayaraMicroImpl
Add-Exports: java.base/jdk.internal.ref jdk.naming.dns/com.sun.jndi.dn
 s java.naming/com.sun.jndi.ldap
Add-Opens: java.base/jdk.internal.loader java.base/java.lang java.base
 /java.net java.base/java.nio java.base/java.util java.base/sun.nio.ch
  java.management/sun.management jdk.management/com.sun.management.int
 ernal java.base/sun.net.www.protocol.jrt java.base/sun.net.www.protoc
 ol.jar java.naming/javax.naming.spi java.rmi/sun.rmi.transport java.l
 ogging/java.util.logging
```

the "Add-Opens" sections is missing _java.base/java.io_

### Expected Outcome

Start Payara Micro successfull

### Current Outcome

Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make private static java.lang.ClassLoader java.io.ObjectInputStream.latestUserDefinedLoader() accessible: module java.base does not "opens java.io" to unnamed module @4bbfd6a

### Reproducer

--

### Operating System

--

### JDK Version

17

### Payara Distribution

Payara Micro

--- a/appserver/extras/payara-micro/payara-micro-distribution/build.xml
+++ b/appserver/extras/payara-micro/payara-micro-distribution/build.xml
@@ -221,7 +221,7 @@
                 <attribute name="Start-Class" value="fish.payara.micro.impl.PayaraMicroImpl"/>
                 <attribute name="Add-Exports" value="java.base/jdk.internal.ref jdk.naming.dns/com.sun.jndi.dns java.naming/com.sun.jndi.ldap"/>
                 <attribute name="Add-Opens"
-                           value="java.base/jdk.internal.loader java.base/java.lang java.base/java.net java.base/java.nio java.base/java.util java.base/sun.nio.ch java.management/sun.management jdk.management/com.sun.management.internal java.base/sun.net.www.protocol.jrt java.base/sun.net.www.protocol.jar java.naming/javax.naming.spi java.rmi/sun.rmi.transport java.logging/java.util.logging"/>
+                           value="java.base/jdk.internal.loader java.base/java.lang java.base/java.net java.base/java.nio java.base/java.util java.base/sun.nio.ch java.management/sun.management jdk.management/com.sun.management.internal java.base/sun.net.www.protocol.jrt java.base/sun.net.www.protocol.jar java.base/java.io java.naming/javax.naming.spi java.rmi/sun.rmi.transport java.logging/java.util.logging"/>
             </manifest>
         </jar>
     </target>

--- a/appserver/extras/payara-micro/payara-micro-distribution/build.xml
+++ b/appserver/extras/payara-micro/payara-micro-distribution/build.xml
@@ -40,7 +40,7 @@
     holder.
 
 -->
-<!-- Portions Copyright 2016-2022 Payara Foundation and/or its affiliates -->
+<!-- Portions Copyright 2016-2023 Payara Foundation and/or its affiliates -->
 
 <project name="payara-micro" default="new.create" basedir=".">
     <property name="rootdir" value="target"/>

